CHANGES.md

```diff
@@ -27,17 +27,69 @@ OpenSSL 3.2
 
  * Added client side support for QUIC
 
-   *Hugo Landau*
+   *Hugo Landau, Matt Caswell, Paul Dale, Tomáš Mráz, Richard Lewitte*
```

Comment:
Not sure who `Richard Lewitte` is...

Reply:
oops, sorry Richard. It happened to me before already.